slashing-protection/src/integration-test/java/tech/pegasys/web3signer/slashingprotection/InterchangeExportIntegrationTest.java

```diff
@@ -142,4 +137,23 @@ void canCreateDatabaseWithEntries() throws IOException, URISyntaxException {
       }
     }
   }
+
+  @Test
+  void failToExportIfGenesisValidatorRootDoesNotExist() throws IOException {
```

Comment:
wondering if its better that we do export it - and put "UNKNOWN" in the field - then again, this really can't happen, and I'd rather know about the problem earlier.